packages/@guardian/source-foundations/src/colour/palette.stories.mdx

```diff
@@ -1,6 +1,6 @@
 import { Meta } from '@storybook/addon-docs';
 import { palette as colours } from './palette';
-import { ColorPalette, ColorItem } from '/.storybook/colorPalette';
+import { ColorPalette, ColorItem } from './storybookColorPalette';
```

Comment:
What's the thinking behind moving these out of the `./storybook` folder?

Reply:
ah - so these are just being used in the `color` and `typography` mdx story respectively but are being included in the global storybook configuration.

the thinking was that if they are only consumed as a set of helper components/methods by those two mdx stories, they should probably live alongside them instead of at the global configuration level

Reply:
My hunch is that this might be one to leave until you move source across to csnx - if `ColorPalette` and `ColorItem` are Storybook specific it makes sense for them to be in the `./storybook` folder. In the new setup there will be a `./storybook` folder per project so would make sense for it to live in there?

Reply:
I'm in two minds about whether they should live in `.storybook` though even if it is just scoped per project. I've always thought that directory was a place for just storybook configuration files to live 🤔 

These feel less like configuration and are more helpers just for those two specific stories, so in my mind they should be local to the two stories that consume them.

Reply:
my feeling is that if they're only used in Storybook they should go in the `.storybook` folder - otherwise you're implying that they're used in the code itself. not a game changer either way though.

Reply:
I see your point there, we called it `storybookColorPalette` to indicate that it belongs to storybook. I suppose individual stories have `.stories.tsx` to differentiate them from just source code, so perhaps we could give the file its own custom extension as well to further strengthen that differentiation?

The thinking in my mind is that we don't locate stories in `.storybook`, so why should we locate their helper methods in there? It feels like we're losing something by not making the two files local to the two stories that they directly support.

Reply:
thats a fair point about .stories files. i was think basically making them local is something that could be done down the line Source is in csnx but not really worth debating too heavily.